test/test_parmed_openmm.py

```diff
@@ -284,3 +284,15 @@ def testWriteXMLParameters(self):
                      provenance=dict(OriginalFile='leaprc.ff14SB',
                      Reference='Maier & Simmerling')
         )
+
+    def testWriteXMLParametersCharmm(self):
+        """ Test writing XML parameter files from Charmm parameter files"""
+
+    params = openmm.OpenMMParameterSet.from_parameterset(
+            charmm.CharmmParameterSet(get_fn('par_all36_prot.prm'),
+                                      get_fn('top_all36_prot.rtf')
+    ))
+    params.write(get_fn('charmm_conv.xml'),
```

Comment:
This should have written=True

Any time a file is written to in the tests it needs to have a file name in the written directory, since that is always cleaned up for future tests.